src/main/java/dev/galacticraft/mod/client/render/MoonSkyProperties.java

```diff
@@ -43,7 +44,7 @@ public Vec3d adjustFogColor(Vec3d color, float sunHeight) {
 
     @Override
     public boolean useThickFog(int camX, int camY) {
-        return false;
+        return MinecraftClient.getInstance().world.getBiome(MinecraftClient.getInstance().cameraEntity.getBlockPos()).getFogColor() == 1447446;
```

Comment:
Is it possible to make this number 0x161616 for readability, or does that break it?

Reply:
![image](https://user-images.githubusercontent.com/26600014/116004780-5607d080-a5d2-11eb-8c53-baf9e573b139.png)
Looks like it should work out